roles/build-release/tasks/build.yaml

```
@@ -8,6 +8,9 @@
     _deps_file: "ansible-{{ antsibull_ansible_version }}.deps"
     _galaxy_file: "ansible-{{ antsibull_ansible_version }}.yaml"
     _build_file: "{{ antsibull_data_dir }}/ansible-{{ antsibull_ansible_version.split('.', 1)[0] }}.build"
+    _tags_file: "{{ '' if antsibull_ansible_version is
+                    _antsibull_packaging_version('7.0.0rc1', '<') else
```

So far no Ansible release included a tags file.
```suggestion
                    _antsibull_packaging_version('7.2.0', '<') else
```